app-text/xsv/xsv-0.13.0.recipe

```diff
@@ -0,0 +1,211 @@
+
+SOURCE_URI_35="https://static.crates.io/crates/threadpool/threadpool-1.7.1.crate"
+CHECKSUM_SHA256_35="e2f0c90a5f3459330ac8bc0d2f879c693bb7a2f59689c1083fc4ef83834da865"
+
+SOURCE_URI_36="https://static.crates.io/crates/ucd-util/ucd-util-0.1.1.crate"
+CHECKSUM_SHA256_36="fd2be2d6639d0f8fe6cdda291ad456e23629558d466e2789d2c3e9892bda285d"
+
+SOURCE_URI_37="https://static.crates.io/crates/unicode-width/unicode-width-0.1.4.crate"
+CHECKSUM_SHA256_37="bf3a113775714a22dcb774d8ea3655c53a32debae63a063acc00a91cc586245f"
+
+SOURCE_URI_38="https://static.crates.io/crates/unicode-xid/unicode-xid-0.1.0.crate"
+CHECKSUM_SHA256_38="fc72304796d0818e357ead4e000d19c9c174ab23dc11093ac919054d20a6a7fc"
+
+SOURCE_URI_39="https://static.crates.io/crates/unreachable/unreachable-1.0.0.crate"
+CHECKSUM_SHA256_39="382810877fe448991dfc7f0dd6e3ae5d58088fd0ea5e35189655f84e6814fa56"
+
+SOURCE_URI_40="https://static.crates.io/crates/utf8-ranges/utf8-ranges-1.0.0.crate"
+CHECKSUM_SHA256_40="662fab6525a98beff2921d7f61a39e7d59e0b425ebc7d0d9e66d316e55124122"
+
+SOURCE_URI_41="https://static.crates.io/crates/void/void-1.0.2.crate"
+CHECKSUM_SHA256_41="6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d"
+
+SOURCE_URI_42="https://static.crates.io/crates/winapi/winapi-0.3.4.crate"
+CHECKSUM_SHA256_42="04e3bd221fcbe8a271359c04f21a76db7d0c6028862d1bb5512d85e1e2eb5bb3"
+
+SOURCE_URI_43="https://static.crates.io/crates/winapi-i686-pc-windows-gnu/winapi-i686-pc-windows-gnu-0.4.0.crate"
+CHECKSUM_SHA256_43="ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6"
+
+SOURCE_URI_44="https://static.crates.io/crates/winapi-x86_64-pc-windows-gnu/winapi-x86_64-pc-windows-gnu-0.4.0.crate"
+CHECKSUM_SHA256_44="712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	xsv = $portVersion
+	cmd:xsv
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cargo$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage xsv$secondaryArchSuffix \
+	$commandBinDir/xsv
+
+BUILD()
+{
+	export CARGO_HOME=$sourceDir/../cargo
+	CARGO_VENDOR=$CARGO_HOME/haiku
+	mkdir -p $CARGO_VENDOR
+	for i in {2..44}; do
+		eval temp=\$sourceDir$i
+		eval shasum=\$CHECKSUM_SHA256_$i
+		pkg=$(basename $temp/*)
+		cp -r $temp/$pkg $CARGO_VENDOR
+		cat <<- EOF > $CARGO_VENDOR/$pkg/.cargo-checksum.json
+		{
+			"package": "$shasum",
+			"files": {}
+		}
+		EOF
+	done
+
+	cat <<- EOF > $CARGO_HOME/config
+	[source.haiku]
+	directory = "$CARGO_VENDOR"
+
+	[source.crates-io]
+	replace-with = "haiku"
+	EOF
+
+	cargo build --release
+}
+
+INSTALL()
+{
+	install -D -m755 target/release/xsv $commandBinDir/xsv 
+	install -D -m644 -t $docDir README.md BENCHMARKS.md
+}
+
+TEST()
+{
+	export CARGO_HOME=\$sourceDir$i/../cargo
```

Comment:
I guess you meant:
```diff
-	export CARGO_HOME=\$sourceDir$i/../cargo
+	export CARGO_HOME=$sourceDir/../cargo
```

Reply:
Whoops; got it confused with the `eval` section. Thanks!